src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -268,29 +268,44 @@ static PyObject *collect(PyObject *self, PyObject *args) {
 
 static bool getEvalOption(PyObject *evalOptions, const char *optionName, const char **s_p) {
   PyObject *value;
-
-  value = PyDict_GetItemString(evalOptions, optionName);
-  if (value)
+  if (PyObject_TypeCheck(evalOptions, &JSObjectProxyType)) {
+    value = PyMapping_GetItemString(evalOptions, optionName);
+  } else {
+    value = PyDict_GetItemString(evalOptions, optionName);
```

Comment:
Why are we not able to use PyMapping_GetItemString for both JSObjectProxy and dict?

Reply:
It just works best this way